### ISSUE TITLE
Website suggestion - instance status checker

### ISSUE THREAD

**Issue by Moire9** [vgskye/e4mc.link#3]
I think it would be nice if there was a way to check if an e4mc instance is active. When I want to see if I can hop on minecraft late at night, in the past I would just use nmap to check if my friend's IPs were open to port 25565. But now that they have switched to using e4mc, it seems that there is no way to do that without booting up minecraft just to check. So if you could add a little page that lets you type in an address to check if it's online, that would be really nice!

**Comment by TwentySeventhLetter** [comment]
https://status.e4mc.link/
Idk if this is exactly what you meant but this exists

**Comment by Moire9** [comment]
No, I mean to check if an individual instance is up. Like, say 5 hours ago my friend started a server `uncle-sense.us.e4mc.link`, a way to check that Uncle Sense is still alive without starting up Minecraft and trying to connect.
